### ISSUE TITLE
Support NX (running npm:nx)

### ISSUE THREAD

**Issue by birkskyum** [denoland/deno#22966]
Version: deno 1.41.3+9c5ddf7

**Platform**

macos sonoma arm64 arm

**Repro**

make new project with deno

`➜ deno run -A npm:create-nx-workspace`

 >  NX   Let's create a new workspace [https://nx.dev/getting-started/intro]

✔ Where would you like to create your workspace? · myorg2
✔ Which stack do you want to use? · none
✔ Package-based monorepo, integrated monorepo, or standalone project? · package-based
✔ Enable distributed caching to make your CI faster · No

try to run it:

```ts
➜ deno run -A npm:nx

 NX   Cannot read properties of undefined (reading 'unref')
 ```

**Comment by bartlomieju** [comment]
Looks like the same problem as https://github.com/denoland/deno/issues/16753#issuecomment-2011193406

**Comment by nathanwhit** [comment]
It looks like this was fixed at some point, trying locally on 2.0.0-rc.2, nx now seems to run without issue.

**Comment by birkskyum** [comment]
That's pretty cool! I guess it's a bit of a minute detail, and not really a change required in this repo either, to have the create-nx-workspace detect and use `deno install` instead of `npm install`, like it does for some other package managers. Probably easier to fix such details after deno 2 is out.
